charts/thoras/templates/api-server/webhooks.yaml

```diff
@@ -49,10 +49,10 @@ webhooks:
     - v1
 - name: "scale.thoras.ai"
   rules:
-  - apiGroups:   ["thoras.ai"]
-    apiVersions: ["v1"]
+  - apiGroups:   ["*"]
+    apiVersions: ["*"]
     operations:  ["UPDATE"]
-    resources:   ["aiscaletargets/scale"]
+    resources:   ["*/scale"]
```

Comment:
I think we decided to be explicit with `deployment` for now, right?

Reply:
Updated to deployments/scale!